data/map planets.txt

```diff
@@ -2504,6 +2550,25 @@ planet Luna
 		threshold 7500
 		fleet "Large Republic" 10
 
+planet "Mordente-Bridi"
+	attributes ccor military station urban
+	landscape land/city6
+	music ambient/machinery
+	description `A testament to the ingenuity of the engineering corps of the now defunct CCOR, the massive Mordente-Bridi Station is one of the largest in human space. Twin cylinders Mordente and Bridi make up the station's hull, built from the strongest alloy the CCOR could manufacture at scale. Bridi Cylinder is pressurized and contains crew quarters and logistical infrastructure amounting to that of a sizable city, while Mordente Cylinder contains what was the Outer Rim's main shipyard.`
```

Comment:
```suggestion
	description `A testament to the ingenuity of the engineering corps of the now defunct Confederated Councils of the Outer Rim, the massive Mordente-Bridi Station is one of the largest in human space. Twin cylinders Mordente and Bridi make up the station's hull, built from the strongest alloy the CCOR could manufacture at scale. Bridi Cylinder is pressurized and contains crew quarters and logistical infrastructure amounting to that of a sizable city, while Mordente Cylinder contains what was the Outer Rim's main shipyard.`
```
I think it makes sense to specify it each time, at least for the first two planets?
there's more on a few other systems I think